R/syntax_test_r.R

```diff
@@ -208,7 +232,7 @@ foo[[bar[1]]] #
 # issue #1120
 sum(x == 1)
 #   ^^^^^^ - variable.parameter.r
-#     ^^ meta.function-call.r meta.function-call.parameters.r keyword.operator.assignment.r
+#     ^^ keyword.operator.assignment.r
```

Comment:
This should be  keyword.operator.comparison.r`

Reply:
https://github.com/sublimehq/Packages/blob/16c9d826244cc265857787d1fb1bd48579837f12/R/R.sublime-syntax#L85-L88

these two scopes should be swapped.

Reply:
Good catch. Just removed the meta scopes from the test case without revealing this bug.

Reply:
Added a commit to fix that and rename the operator scopes according to other syntaxes.